gui/default/syncthing/core/notifications.html

```diff
@@ -116,3 +116,28 @@ <h3 class="panel-title"><span class="fas fa-bolt"></span>&ensp;<span translate>A
     </div>
   </div>
 </notification>
+
+<notification id="authenticationUserAndPassword">
+  <div class="panel panel-success">
+    <div class="panel-heading">
+      <h3 class="panel-title"><span class="fas fa-bolt"></span>&nbsp;<span translate>GUI Authentication: Set User and Password</span></h3>
+    </div>
+    <div class="panel-body">
+      <p>
+      <span translate>Username/Password has not been set for the GUI authentication. Please consider setting it up.</span>
```

Comment:
Indentation (not commenting on more instances).

Reply:
Fixed, hope I found them all